Lib/dis.py

```diff
@@ -107,13 +112,19 @@ def pretty_flags(flags):
     return ", ".join(names)
 
 def _get_code_object(x):
-    """Helper to handle methods, functions, generators, strings and raw code objects"""
+    """Helper to handle methods, functions, generators,
```

Comment:
Even though this is a "private" method, it should follow the guidelines for docstings. The first line should be a complete sentence shorter than 72 chars describing the function.